src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -44,6 +51,9 @@ object ConsignmentFields {
       Field("consignmentid", OptionType(UuidType), resolve = _.value.consignmentid),
       Field("userid", UuidType, resolve = _.value.userid),
       Field("seriesid", UuidType, resolve = _.value.seriesid),
+      Field("dateTime", OptionType(LocalDateTimeType), resolve = _.value.dateTime.get.toLocalDateTime),
+      Field("transferInitiatedDatetime", OptionType(LocalDateTimeType), resolve = _.value.transferInitiatedDatetime.get.toLocalDateTime),
+      Field("exportDatetime", OptionType(LocalDateTimeType), resolve = _.value.exportDatetime.get.toLocalDateTime),
       Field(
```

Comment:
What happens here if the `transferInitiatedDatetime` or `exportDatetime` aren't set? It looks like the call to `.get` would fail. Would it be better to do something like `.map(_.toLocalDateTime)`

Reply:
There's an error thrown in by the query, and the value is populated with null: 

"errors": [
        {
            "message": "Internal server error",
            "path": [
                "getConsignment",
                "transferInitiatedDatetime"
            ],
            "locations": [
                {
                    "column": 9,
                    "line": 5
                }
            ]
        }

Reply:
That's not what we want though is it? We want it to return the consignment with the dates empty instead of an error.

Reply:
Will use map instead